torch/_dynamo/variables/distributed.py

```diff
@@ -152,6 +152,34 @@ def var_getattr(self, tx, name: str) -> VariableTracker:
             return ConstantVariable.create(self.value.ndim)
         return super().var_getattr(tx, name)
 
+    def call_method(
+        self,
+        tx,
+        name,
+        args: "List[VariableTracker]",
+        kwargs: "Dict[str, VariableTracker]",
+    ) -> "VariableTracker":
+        if name == "size":
+            mesh_dim_var = None
+            if len(args) == 1:
```

Comment:
Better to do more explicit/complete checking of args/kwargs- raise unsupported if getting unexpected combinations and try to support all the valid combinations (or if skipping some make a comment saying why)